prog/probe.go

```diff
@@ -152,7 +152,7 @@ func probeMain(flags probeFlags) {
 		defer dnsSnooper.Stop()
 	}
 
-	endpointReporter := endpoint.NewReporter(hostID, hostName, flags.spyProcs, flags.useConntrack, flags.procEnabled, flags.procRoot, scanner, dnsSnooper)
+	endpointReporter := endpoint.NewReporter(hostID, hostName, flags.spyProcs, flags.useConntrack, flags.procEnabled, flags.procRoot, flags.conntrackBufferSize, scanner, dnsSnooper)
```

Comment:
Time for a config datastructure? (the number of parameters is getting out of hand)